src/content/docs/usage/workspaces.mdx

```diff
@@ -29,18 +39,25 @@ You can create a Workspace.
    daytona create
    ```
 
-2. Select the Git Provider and repository you want your Workspace to be configured with.
-3. Enter a name for your Workspace.
+2. Select the Git Provider and the repository you want your Workspace to be configured with. This selection links your Workspace directly to your version control system.
+
+   ```text
+      Choose a Provider
+      2 items
+      ===
+      GitHub
+      Enter a custom repository URL
+   ```
+
+3. Enter a name for your Workspace. Provide a descriptive name for your Workspace that reflects its purpose or project.
    <Aside type="note">
      You can press `F10` to configure advanced settings for your Workspace,
      including setting the [Builder](../usage/builders).
    </Aside>
-4. Wait for Daytona to create the Workspace.
+4. Wait while Daytona sets up your Workspace. It handles all the initialization and configuration of your environment.
```

Comment:
@stefanicjuraj i like your example above of cli output : 
```text
      Choose a Provider
      2 items
      ===
      GitHub
      Enter a custom repository URL
   ```

so doing the same here might make sense so the user knows what a correct created WS screen looks like